modules/cnf-fast-event-notifications-api-refererence.adoc

```diff
@@ -237,7 +237,7 @@ $ oc logs -f linuxptp-daemon-cvgr6 -n openshift-ptp -c cloud-event-proxy
             "resource":"/sync/sync-status/os-clock-sync-state",
             "dataType":"notification",
             "valueType":"enumeration",
-            "value":"LOCKED"
+            "value":"SYNCHRONIZED"
```

Comment:
@amolnar-rh Am I reading this right? This is an `os-clock-sync-state` notification, for which _valid states are `LOCKED` or `FREERUN`_, and not `SYNCHRONIZED`??

Reply:
Same with your other [PR for 4.14](https://github.com/openshift/openshift-docs/pull/91340/files#diff-9907de885cc5d8e205a9d02786362bea1e270aac316fef8006d986acb55b7d49R240)?